libnavigation-core/src/test/java/com/mapbox/navigation/core/internal/formatter/MapboxDistanceFormatterTest.kt

```diff
@@ -308,4 +308,30 @@ class MapboxDistanceFormatterTest {
 
         assertEquals("181 ft", result.toString())
     }
+
+    @Config(qualifiers = "en")
+    @Test
+    fun formatDistanceZero() {
+        val result = MapboxDistanceFormatter(
+            DistanceFormatterOptions.Builder(ctx)
+                .unitType(IMPERIAL)
+                .roundingIncrement(5)
```

Comment:
For consistency, should we add `INCREMENT_FIVE` and `INCREMENT_TWO` to `Rounding`?

Reply:
I don't think there's anything special about those constants. The test is demonstrating any value for the rounding increment can be used.  It's not enforcing any specific business rule regarding those specific values.